experimental/AlgebraicStatistics/src/GaussianGraphicalModels.jl

```diff
@@ -4,6 +4,8 @@
 #
 ###################################################################################
 
+export ci_ideal, ci_polynomial, ci_structure
```

Comment:
let's keep all exports in the AlgebraicsStatistics.jl at least for now. this will be easier if we have to move them all later.